apps/sports-helsinki/src/domain/search/venueSearch/VenueSearch.tsx

```diff
@@ -7,14 +7,14 @@ import {
   useCommonTranslation,
   MultiSelectDropdown,
   getUrlParamAsArray,
+  // eslint-disable-next-line import/no-unresolved
```

Comment:
This doesn't look right. This should not be needed. `events-helsinki-components` is very much needed and it must be resolvable.